README.md

```diff
@@ -25,6 +25,10 @@ Docker must be installed to build and run code (For Google workstations, see
 go/installdocker.). Ruby is required to run our development scripts, which
 document common operations and provide a convenient way to perform them.
 
+Add `export UID` to your `.bashrc`. All our Docker commands run as your user,
```

Comment:
I would prefer that we do not recommend doing this. It could hide bugs that we might otherwise fix.